docs/API.md

```diff
@@ -1287,37 +1287,28 @@ __Parameters__
 __Example 1__
 
 ```js
-minioClient.removeObject('mybucket', 'photo.jpg', function(err) {
-  if (err) {
-    return console.log('Unable to remove object', err)
-  }
-  console.log('Removed the object')
-})
+await minioClient.removeObject('mybucket', 'photo.jpg')
+console.log('Removed the object')
```

Comment:
When we directly copy, paste and run this function in an environment where the MinIO Client has been created, we get a syntax error essentially saying we should also use the `async` keyword.

```
SyntaxError: await is only valid in async functions and the top level bodies of modules
```

Instead, for the ease of use, should we have something like the following here?
```suggestion
(async function () {
  await minioClient.removeObject("mybucket", "photo.jpg");
  console.log("Removed the object");
})();
```